news.html

```diff
@@ -17,7 +17,9 @@ <h5 class="mb-0">
              href="https://docs.dockstore.org/en/stable/getting-started/getting-started-with-notebooks.html">
               Notebooks</a>
       </h5>
-        <span>Learn how to register notebooks in Dockstore</span>
+        <span>Learn how to register notebooks in Dockstore. Explore an <a target="_blank" rel="noopener noreferrer"
```

Comment:
Maybe link the two sentences or transition in some way, like "as a fun example" ...

Reply:
Basically don't want users to confuse the TRS example with learning how to register (in the header above)